mapdata/WarcraftLegacies/UpgradeData/Core/1735222610.json

```diff
@@ -32,6 +32,10 @@
       "Id": 829386087,
       "Type": 2,
       "Value": 1
+    },
+    {
+      "Id": 1835820903,
+      "Value": -45
```

Comment:
What does this change do? Not mentioned in the PR description I think.

Reply:
This is for huntress second glaive upgrade being cheaper

Reply:
Interesting - traditionally researches have increasing costs to produce diminishing returns. Otherwise you end up in the situation where it never makes sense to get level 1 unless you also intend to get level 2. Won't block it though.